eng/Subsets.props

```diff
@@ -204,11 +204,13 @@
                              $(CoreClrProjectRoot)tools\r2rtest\R2RTest.csproj;
                              $(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2.csproj" Category="clr" />
 
+    <ProjectToBuild Condition="'$(TargetArchitecture)' != 'x64'" Include="$(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2_crossarch.csproj" Category="clr" />
```

Comment:
This assumes that everyone only builds on x64. That breaks all ports to all other platforms that are doing non-cross builds...

Reply:
Thank you for reporting this issue. I think the condition here and 6 lines below that line should be `'$(TargetArchitecture)' != 'x64' and '$(BuildArchitecture)' == 'x64'`.  I'll submit a PR after verification.